### PR TITLE
Update manual-backup.md

### PR DESCRIPTION
**Pull request by rsignell**
Adding manual backup instructions for Azure

### CODE REVIEW

**Comment by rsignell** [comment]
Perhaps @Adam-D-Lewis for review?

**Comment by viniciusdc** [comment]
Hi @rsignell, thanks for opening the PR!! Can you give me a bit of context about where this was used?

**Comment by rsignell** [comment]
unfortunately I cannot!

**Comment by viniciusdc** [comment]
> unfortunately I cannot!

No worries, thanks for opening the PR 🚀

**Comment by Adam-D-Lewis** [comment]
I'll take a look today

File: docs/docs/how-tos/manual-backup.md [hunk 1]
```diff
@@ -175,6 +175,27 @@ gsutil cp 2021-04-23.tar gs://<your_bucket_name>/backups/2021-04-23.tar
 cd /data
 gsutil cp gs://<your_bucket_name>/backups/2021-04-23.tar .
 ```
+### Azure
+# install Azure CLI
+curl -sL https://aka.ms/InstallAzureCLIDeb |  bash
+az login --use-device-code
```
Comment (kcpevey) [inline docs/docs/how-tos/manual-backup.md:181]: I think the CI failures might resolve if we add code block quotes around these and the codes below

**Comment by Adam-D-Lewis** [comment]
Thanks for this @rsignell!  I branched off and updated some formatting issues.  Closing in favor of https://github.com/nebari-dev/nebari-docs/pull/494.  Feel free to comment over there with any additional input!

**Pull request closed without merging**